sentinel-core/src/test/java/com/alibaba/csp/sentinel/config/SentinelConfigTest.java

```diff
@@ -97,12 +110,12 @@ public void testLoadProperties() throws IOException {
             out.flush();
             out.close();
 
-            Assert.assertTrue(SentinelConfig.getConfig(CHARSET).equals("utf-8"));
-            Assert.assertTrue(SentinelConfig.getConfig(SINGLE_METRIC_FILE_SIZE).equals("1000"));
-            Assert.assertTrue(SentinelConfig.getConfig(TOTAL_METRIC_FILE_COUNT).equals("20"));
-            Assert.assertTrue(SentinelConfig.getConfig(COLD_FACTOR).equals("5"));
-            Assert.assertTrue(SentinelConfig.getConfig(STATISTIC_MAX_RT).equals("1000"));
-            Assert.assertTrue(SentinelConfig.getAppName().equals("sentinel_test"));
+            assertEquals("utf-8", getConfig(CHARSET));
+            assertTrue(SentinelConfig.getConfig(SINGLE_METRIC_FILE_SIZE).equals("1000"));
+            assertTrue(SentinelConfig.getConfig(TOTAL_METRIC_FILE_COUNT).equals("20"));
+            assertTrue(SentinelConfig.getConfig(COLD_FACTOR).equals("5"));
+            assertTrue(SentinelConfig.getConfig(STATISTIC_MAX_RT).equals("1000"));
+            assertTrue(SentinelConfig.getAppName().equals("sentinel_test"));
```

Comment:
Why change `Assert.assertTrue(SentinelConfig.getConfig(CHARSET).equals("utf-8"));` to `assertEquals("utf-8", getConfig(CHARSET));`?  it does function well but looks not the same style as the following lines.

Reply:
> Why change `Assert.assertTrue(SentinelConfig.getConfig(CHARSET).equals("utf-8"));` to `assertEquals("utf-8", getConfig(CHARSET));`? it does function well but looks not the same style as the following lines.

I have changed it back, this maybe changed by my plugin.

Reply:
Okay, I've also encountered situations where the IDE sometimes prompts for automatic optimization.